flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogDataTypeTest.java

```diff
@@ -177,7 +178,7 @@ public void testComplexDataTypes() throws Exception {
 			DataTypes.ROW(
 				DataTypes.FIELD("0", DataTypes.BOOLEAN()),
 				DataTypes.FIELD("1", DataTypes.BOOLEAN()),
-				DataTypes.FIELD("2", DataTypes.DATE())),
+				DataTypes.FIELD("2", HiveTypeUtil.HIVE_DATE_TYPE)),
```

Comment:
This seems not very user friendly because now users are supposed to know this extra static member HiveTypeUtil.HIVE_DATE_TYPE besides DataTypes